dashboard-react.js

```diff
@@ -249,77 +249,112 @@ NETDATA.statistics = {
 
 NETDATA.themes = {
   white: {
-    bootstrap_css: 'css/bootstrap-3.3.7.css',
-    dashboard_css: 'css/dashboard.css?v20180210-1',
-    background: '#FFFFFF',
-    foreground: '#000000',
-    grid: '#F0F0F0',
-    axis: '#F0F0F0',
-    highlight: '#F5F5F5',
-    colors: ['#3366CC', '#DC3912', '#109618', '#FF9900', '#990099', '#DD4477',
-      '#3B3EAC', '#66AA00', '#0099C6', '#B82E2E', '#AAAA11', '#5574A6',
-      '#994499', '#22AA99', '#6633CC', '#E67300', '#316395', '#8B0707',
-      '#329262', '#3B3EAC'],
-    easypiechart_track: '#f0f0f0',
-    easypiechart_scale: '#dfe0e0',
-    gauge_pointer: '#C0C0C0',
-    gauge_stroke: '#F0F0F0',
+    bootstrap_css: "css/bootstrap-3.3.7.css",
+    dashboard_css: "css/dashboard.css?v20180210-1",
+    background: "#FFFFFF",
+    foreground: "#000000",
+    grid: "#F0F0F0",
+    axis: "#F0F0F0",
+    highlight: "#F5F5F5",
+    colors: [
+      "#B0E952",
+      "#FC8D5E",
+      "#3366CC",
+      "#DC3912",
+      "#109618",
+      "#FF9900",
+      "#990099",
+      "#DD4477",
+      "#3B3EAC",
+      "#66AA00",
+      "#0099C6",
+      "#B82E2E",
+      "#AAAA11",
+      "#5574A6",
+      "#994499",
+      "#22AA99",
+      "#6633CC",
+      "#E67300",
+      "#316395",
+      "#8B0707",
+      "#329262",
+      "#3B3EAC",
+    ],
+    easypiechart_track: "#f0f0f0",
+    easypiechart_scale: "#dfe0e0",
+    gauge_pointer: "#93A3B0",
+    gauge_stroke: "#ECEFF2",
     gauge_gradient: false,
-    gauge_stop_color: '#FC8D5E',
-    gauge_start_color: '#B0E952',
+    gauge_stop_color: "#FF4C00",
+    gauge_start_color: "#FF4C00",
```

Comment:
Please verify that this is not a mistake here. It seems weird to use the same color.

Reply:
We might have to get some input from Andrzej on this one! The design doesn't include a gradient, so I took it for granted that it should be that way

![image](https://user-images.githubusercontent.com/8643883/98276601-ddb73f80-1f9e-11eb-94db-66960670f781.png)

Reply:
@novykh Andrzej just verified it. Flat colors only he said. So I suppose we go with this...